### ISSUE TITLE
duplicates for golden helipad

### ISSUE THREAD

**Issue by DonnaHaga** [bcgov/envair#18]
R Version 4.3.1 envair version 0.4.0.100.

```
envair::importBC_data(parameter_or_station="pm25",years=2021) %>% 
                            dplyr::filter(STATION_NAME=="Golden Helipad")
```

Contains duplicates as follows. Data is identical but has both level 0 and level 2 validation status listed

![duplicates](https://github.com/bcgov/envair/assets/24443567/ad3d24ae-c47b-4455-800a-156830f5c1e0)

**Comment by ZoeGao218** [comment]
The issue of the duplicate entries has been fixed in the latest version: 0.4.0.300.

**Comment by DonnaHaga** [comment]
Great!
